### PR TITLE
Feature/estefania sora accessible form

### PR DESCRIPTION
**Pull request by EstefaSora125**
New accesible form project

### CODE REVIEW

File: 1-the-basics/index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        <li>Read</li>
+        <li>Watch movies<li>
+    </ul>
+    <p>I'm a  <strong></strong>grumy</p>
+    <img height="400" width="500" src="https://s3.amazonaws.com/rtvc-assets-senalcolombia.gov.co/s3fs-public/field/image/mi-vecino-totoro-mei-satsuke-lluvia-parada-bus.jpg"alt="My new image ">
+    <br>
+    <img height="400" width="300" src="./butterfly.jpg">
+
+</body>
+</html>
```
Comment (juancho11gm) [inline 1-the-basics/index.html]: This file is for the first exercise. Please remove it. Remember to create the new branch from `master` branch.

File: 5-Accessibility/accessible-form.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <title>Form</title>
+</head>
+<body>
+    <main>
+        <form>
+            <fieldset>
+                <legend>Please enter your contact details</legend>
+                    <label for="user_name">Your name
+                    <input type="text" id="user_name" name="name" required>
+                    <abbr title="required" aria-label="required">*</abbr>
```
Comment (juancho11gm) [inline 5-Accessibility/accessible-form.html:15]: nice to see the `aria-attributes`

File: 5-Accessibility/accessible-form.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                 <input type="checkbox" id="taste_3" name="taset_fullstack" value="fullstack">
+                 <label for="taste_3">Fullstack Developer</label>
+                 <input type="checkbox" id="taste_4" name="taste_other" value="other">
+                 <label for="taste_4">Other</label>
+             </fieldset>
+            <label for="text-box">Please enter a message</label>
+            <div>
+                <textarea name="message" id="text-box" cols="40" rows="10"></textarea>
+            </div>
+            <button type="submit">Send mesagge</button>
```
Comment (juancho11gm) [inline 5-Accessibility/accessible-form.html]:
```suggestion
            <button>Send mesagge</button>
```

**Review by juancho11gm — CHANGES_REQUESTED** [review]
Nice job Estefanía! We are seeing your progress. Please see the comments and send the changes.

**Review by pushdev-code — APPROVED** [review]

**Comment by pushdev-code** [comment]
Please resolve conflicting files in this PR. `1-the-basics/index.html` should not have any change.

**Comment by EstefaSora125** [comment]
Change the project branch

**Pull request closed without merging**